tests/unit/constraints/test_tabular.py

```diff
@@ -2744,594 +1233,483 @@ def test_reverse_transform_datetime_drop_high(self):
-            - apply an exponential to the input
-            - subtract 1
-            - subtract from the high column
-            - convert the output to integers
-            - add back the dropped column
+class TestScalarInequality():
+
+    def test__validate_inputs_incorrect_column(self):
+        """Test the ``_validate_inputs`` method.
+
+        Ensure the method crashes when the column name is not a string.
 
-        Setup:
-        - ``_drop`` is set to ``low``.
         Input:
-        - Table with a diff column that contains the constant np.log(4).
-        Output:
-        - Same table with the low column replaced by the high one - 3, as int
-        and the diff column dropped.
+        - a non-string
+        - a number
+        - an inequality
+        Side effect:
+        - Raise ``ValueError`` because the column name must be a string
         """
-        # Setup
-        instance = GreaterThan(low='a', high='b', strict=True, drop='low')
-        instance._dtype = [pd.Series([1]).dtype]    # exact dtype (32 or 64) depends on OS
-        instance._diff_columns = ['a#b']
-        instance._columns_to_reconstruct = ['a']
+        # Run / Assert
+        with pytest.raises(ValueError):
+            ScalarInequality._validate_inputs(column_name=['a'], value=1, relation='>')
+
+    def test__validate_inputs_incorrect_value(self):
+        """Test the ``_validate_inputs`` method.
+
+        Ensure the method crashes when the value is not numerical.
+
+        Input:
+        - a string
+        - a non-number
+        - an inequality
+        Side effect:
+        - Raise ``ValueError`` because the value must be a numerical
+        """
+        # Run / Assert
+        with pytest.raises(ValueError):
+            ScalarInequality._validate_inputs(column_name='a', value='b', relation='>')
+
+    def test__validate_inputs_incorrect_relation(self):
+        """Test the ``_validate_inputs`` method.
+
+        Ensure the method crashes when the column name is not a string.
+
+        Input:
+        - a string
+        - a number
+        - a non-inequality
+        Side effect:
+        - Raise ``ValueError`` because the relation must be an inequality
+        """
+        # Run / Assert
+        with pytest.raises(ValueError):
+            ScalarInequality._validate_inputs(column_name='a', value=['b', 'c'], relation='=')
+
+    @patch('sdv.constraints.tabular.ScalarInequality._validate_inputs')
+    def test___init___(self, mock_validate):
+        """Test the ``ScalarInequality.__init__`` method.
 
+        The passed arguments should be stored as attributes.
+
+        Input:
+        - low_column_name = 'a'
+        - high_column_name = 'b'
+        - relation = '>'
+        Side effects:
+        - instance._column_name = 'a'
+        - instance._value = 1
+        - instance._diff_column_name = 'a#'
+        - instance._operator = np.greater
+        - instance._dtype = None
+        - instance._is_datetime = None
```

Comment:
Same thing here. Let's describe why the parameters are being set to these values rather than listing them